Easydict/Swift/Service/Volcano/VolcanoService.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  VolcanoService.swift
+//  Easydict
+//
+//  Created by Jerry on 2024-08-11.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Alamofire
+import Defaults
+import Foundation
+
+@objc(EZVolcanoService)
+public final class VolcanoService: QueryService {
+    // MARK: Public
+
+    override public func serviceType() -> ServiceType {
+        .volcano
+    }
+
+    override public func link() -> String? {
+        "https://translate.volcengine.com"
+    }
+
+    override public func name() -> String {
+        NSLocalizedString("volcano_translate", comment: "The name of Volcano Translate")
+    }
+
+    override public func supportLanguagesDictionary() -> MMOrderedDictionary<AnyObject, AnyObject> {
+        let orderedDict = MMOrderedDictionary<AnyObject, AnyObject>()
+        for (key, value) in VolcanoTranslateType.supportLanguagesDictionary {
+            orderedDict.setObject(value as NSString, forKey: key.rawValue as NSString)
+        }
+        return orderedDict
+    }
+
+    override public func ocr(_: EZQueryModel) async throws -> EZOCRResult {
+        logInfo("Volcano Translate currently does not support OCR")
+        throw QueryServiceError.notSupported
+    }
+
+    override public func translate(
+        _ text: String,
+        from: Language,
+        to: Language,
+        completion: @escaping (EZQueryResult, Error?) -> ()
+    ) {
+        let result = result
+        // swiftlint:disable line_length
+        guard !accessKeyID.isEmpty else {
+            let noAccessKeyIDError = EZError(
+                type: .missingAPIKey,
+                description: "Missing Volcano AccessKeyID. Volcano Service requires users' own API Key. Get it at https://www.volcengine.com"
+            )
+            completion(result, noAccessKeyIDError)
+            return
+        }
+        guard !secretAccessKey.isEmpty else {
+            let noSecretAccessKey = EZError(
+                type: .missingAPIKey,
+                description: "Missing Volcano SecretAccessKey. Volcano Service requires users' own API Key. Get it at https://www.volcengine.com"
+            )
+            completion(result, noSecretAccessKey)
+            return
+        }
```

Comment:
Please try to reduce duplicate code, for example, you can optimize it as follows:

```swift
func validateAPIKey(_ key: String, keyType: String) -> EZError? {
    if key.isEmpty {
        return EZError(
            type: .missingAPIKey,
            description: "Missing Volcano \(keyType). Volcano Service requires users' own API Key. Get it at https://www.volcengine.com"
        )
    }
    return nil
}

if let error = validateAPIKey(accessKeyID, keyType: "AccessKeyID") {
    completion(result, error)
    return
}

if let error = validateAPIKey(secretAccessKey, keyType: "SecretAccessKey") {
    completion(result, error)
    return
}
```

Reply:
After this modification, there should be no need to temporarily disable swiftlint anymore.

Reply:
Completed with https://github.com/tisfeng/Easydict/pull/636/commits/a8224bcc122693224a3d4b149c3bba762c846437